tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -230,7 +233,7 @@ func (c *Reconciler) reconcile(ctx context.Context, run *v1alpha1.Run, status *p
 	}
 
 	// Determine how many iterations of the Task will be done.
-	totalIterations, err := computeIterations(run, pipelineLoopSpec)
+	totalIterations, interationElements, err := computeIterations(run, pipelineLoopSpec)
```

Comment:
There's many `interationElements`, I guess `interation` should be `iteration`?

Reply:
you are right, I corrected the typo.